### DISCUSSION TITLE
Ability to track metrics (no target)

### DISCUSSION THREAD

**Discussion by Level3Manatee** [iSoron/uhabits#1553] [Feature Requests]
**Feature request**
I would like to track metrics without a target, so that I can later see them in relation to the habits I tracked in a given time period.

Use cases could be:
- daily mood, health metrics like bodily well-being, headache score or similar things for chronic conditions
- any habit that might not have a clear target or an intended behavioral change
- circumstances that can be measured which might factor into habits

It could be useful to see these things in relation to one another, for example to see if the mood changes when certain habits are observed, if some habits should be changed, or in general to track things that just "are" without a clear goal, to provide some context for the other metrics.

**Comment by hiqua** [comment]
I guess you could just use 0 as a target?

For advanced data analysis I think it's better to just export the data as a .csv and do that in whatever tool you prefer.

**Reply by Level3Manatee** [reply]
Thanks. 0 might work, that didn't occur to me. Maybe a target type of 'none' would make that more obvious from a usability standpoint?

As for data analysis, comparing curves should be enough to see trends, so getting a gist should be possible right now, no? I only installed it today, so I'm not familiar with anything past the screenshots.

It might be worthwhile visualizing curves for multiple habits at once if that isn't possible already.

**Reply by hiqua** [reply]
I think it gets quite complicated really quickly for something that's really personal in terms of what you're interested in, and that's simple enough to do for most people that are actually interested in that.

**Comment by b4tg5s** [comment]
I am also interested in this feature.

Beeing able to track activities without the urge that arises from an explicit target can provide beneficial insights in behavior/habits and allows to calmly observe instead of extrinsically motivate. Sometimes I want just want the information without the feeling of achievement or reward that comes from reaching a target.

For example: I want an explicit target on exercise which is extrinsically motivated while I only want the tracking information on meditation which I don't want to be related to external goals.

Setting the target to 0 does not work because it defaults the setting to "each day" in my case.
